### PR TITLE
Change template paths to adapt to the convention

### PR DESCRIPTION
**Pull request by axzx**
I am targeting this branch, because this is BC.

## Changelog

```markdown
### Changed
- template paths to adapt to the convention
```

## Subject

Changes in this PR were created to override templates according to new convention which is described below:

[How to Override Templates from Third-Party Bundles](https://symfony.com/doc/current/templating/overriding.html)

e.g.:

old way: `SonataAdminBundle::standard_layout.html.twig`
new way: `@SonataAdmin/standard_layout.html.twig`

### CODE REVIEW

**Comment by jordisala1991** [comment]
Can you explain better why this change is needed and improve commit message and changelog? end users need to understand it also

**Comment by axzx** [comment]
@greg0ire What can I do to make the tests pass?

File: docs/reference/action_create_edit.rst [hunk 1]
```diff
@@ -29,8 +29,8 @@ SonataAdmin Options that may affect the create or edit view:
             form_type:         standard # can also be 'horizontal'
 
         templates:
-            edit:              SonataAdminBundle:CRUD:edit.html.twig
-            tab_menu_template: SonataAdminBundle:Core:tab_menu_template.html.twig
+            edit:              @SonataAdmin/CRUD/edit.html.twig
+            tab_menu_template: @SonataAdmin/Core/tab_menu_template.html.twig
```
Comment (greg0ire) [inline docs/reference/action_create_edit.rst]: You have to quote these because of the `@`

File: docs/reference/action_list.rst [hunk 2]
```diff
@@ -20,16 +20,16 @@ SonataAdmin Options that may affect the list view:
...
+            list:                       @SonataAdmin/CRUD/list.html.twig
+            action:                     @SonataAdmin/CRUD/action.html.twig
+            select:                     @SonataAdmin/CRUD/list__select.html.twig
+            list_block:                 @SonataAdmin/Block/block_admin_list.html.twig
+            short_object_description:   @SonataAdmin/Helper/short-object-description.html.twig
+            batch:                      @SonataAdmin/CRUD/list__batch.html.twig
+            inner_list_row:             @SonataAdmin/CRUD/list_inner_row.html.twig
+            base_list_field:            @SonataAdmin/CRUD/base_list_field.html.twig
+            pager_links:                @SonataAdmin/Pager/links.html.twig
+            pager_results:              @SonataAdmin/Pager/results.html.twig
```
Comment (greg0ire) [inline docs/reference/action_list.rst]: Here too

File: docs/reference/configuration.rst [hunk 3]
```diff
@@ -138,38 +138,38 @@ Full Configuration Options
...
+                batch_confirmation:   '@SonataAdmin/CRUD/batch_confirmation.html.twig'
+                inner_list_row:       '@SonataAdmin/CRUD/list_inner_row.html.twig'
+                outer_list_rows_mosaic:  '@SonataAdmin/CRUD/list_outer_rows_mosaic.html.twig'
+                outer_list_rows_list:  '@SonataAdmin/CRUD/list_outer_rows_list.html.twig'
+                outer_list_rows_tree:  '@SonataAdmin/CRUD/list_outer_rows_tree.html.twig'
+                base_list_field:      '@SonataAdmin/CRUD/base_list_field.html.twig'
+                pager_links:          '@SonataAdmin/Pager/links.html.twig'
+                pager_results:        '@SonataAdmin/Pager/results.html.twig'
+                tab_menu_template:    '@SonataAdmin/Core/tab_menu_template.html.twig'
+                knp_menu_template:    '@SonataAdmin/Menu/sonata_menu.html.twig'
```
Comment (greg0ire) [inline docs/reference/configuration.rst]: Everywhere there is yaml, in fact

**Comment by axzx** [comment]
https://github.com/sonata-project/SonataAdminBundle/commit/8088e9384c85689d016dcc587f724a653388a290

**Comment by jordisala1991** [comment]
Can you rebase instead of adding a new commit? otherwise we won't be able to merge your PR

**Comment by jordisala1991** [comment]
To rebase you need to:

`git rebase -i upstream/3.x` (if upstream is the remote that points to the original SonataAdminBundle)
Fix conflicts when they show up
`git rebase --continue`
`git push origin patch-3` (if origin is the remote of your fork)

@greg0ire has a better explanation on how to do a rebase, but this is what I follow

**Comment by jordisala1991** [comment]
Ping @sonata-project/contributors

**Comment by greg0ire** [comment]
Thanks @axzx !

**Pull request merged**